web/package.json

```diff
@@ -31,6 +31,7 @@
     "react-dom": "^18",
     "react-hook-form": "^7.53.2",
     "react-icons": "^5.3.0",
+    "react-router-dom": "^7.0.2",
```

Comment:
```suggestion
```

we are using nextjs, in nextjs `react-router-dom` is not needed - the paths are defined based on folder structure